runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -467,10 +468,36 @@ void takeScreenshot(String fileName, boolean displayDate)
 				}
 			}
 
+			try
+			{
+				World world = new WorldClient().lookupWorlds().findWorld(client.getWorld());
```

Comment:
This should definitely be determined upon login and not here.

Reply:
You are correct and this have been adjusted.